conda_smithy/lint_recipe.py

```diff
@@ -489,6 +489,16 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
                             )
                         )
 
+    # 24: make sure that license, license_file, and license_family are all present
+    license_fields = ["license", "license_file", "license_family"]
```

Comment:
Why is `license_family` required? This field makes no sense for packages that are dual licensed.

Reply:
They can be OTHER in that case

Reply:
Then, I'd prefer that the `license_family` field was changed to a hint.

Reply:
Ok, as long as we block on staged-recipes for it